vumi_twilio_api/twilio_api.py

```diff
@@ -149,10 +149,25 @@ def _handle_connected_call(
         self.session_manager.save_session(session_id, session)
         twiml = yield self._get_twiml_from_client(session)
         for verb in twiml:
-            self._handle_twiml_verb(verb)
+            yield self._handle_twiml_verb(verb, session)
 
-    def _handle_twiml_verb(self, verb):
-        pass
+    @inlineCallbacks
+    def _handle_twiml_verb(self, verb, session):
+        if not verb:
+            return
+
+        if verb.name == "Play":
+            yield self.send_to(
+                session['To'], '',
```

Comment:
I think it's better to set content to ``None`` here. It's not that the content is blank -- the Play verb simply doesn't give us any content.